### PR TITLE
[zh-cn]: sync translation for `FileReader.{abort(), abort, load}`

### PR DESCRIPTION
**Pull request by skyclouds2001**
### Description

see https://developer.mozilla.org/en-US/docs/Web/API/FileReader/abort and https://developer.mozilla.org/en-US/docs/Web/API/FileReader/abort_event and https://developer.mozilla.org/en-US/docs/Web/API/FileReader/load_event

### Related issues and pull requests

depends on https://github.com/mdn/content/pull/33568
part of the https://github.com/mdn/translated-content/issues/20165

### CODE REVIEW

File: files/zh-cn/web/api/filereader/abort_event/index.md [hunk 1]
```diff
@@ -23,42 +23,42 @@ onabort = (event) => {};
...
+    <label for="avatar">选择个人资料图片：</label>
     <input
       type="file"
       id="avatar"
       name="avatar"
       accept="image/png, image/jpeg" />
   </div>
 
-  <img src="" class="preview" height="200" alt="图片预览" />
+  <img src="" class="preview" height="200" alt="Image preview" />
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/filereader/abort_event/index.md:58]:
```suggestion
  <img src="" class="preview" height="200" alt="图片预览" />
```

File: files/zh-cn/web/api/filereader/load_event/index.md [hunk 2]
```diff
@@ -1,55 +1,65 @@
...
+    <label for="avatar">选择个人资料图片：</label>
     <input
       type="file"
       id="avatar"
       name="avatar"
       accept="image/png, image/jpeg" />
   </div>
 
-  <img src="" class="preview" height="200" alt="Image preview..." />
+  <img src="" class="preview" height="200" alt="Image preview" />
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/filereader/load_event/index.md:58]:
```suggestion
  <img src="" class="preview" height="200" alt="图片预览" />
```

File: files/zh-cn/web/api/filereader/load_event/index.md [hunk 3]
```diff
@@ -104,8 +111,7 @@ const eventLog = document.querySelector(".event-log-contents");
 const reader = new FileReader();
 
 function handleEvent(event) {
-  eventLog.textContent =
-    eventLog.textContent + `${event.type}: ${event.loaded} bytes transferred\n`;
+  eventLog.textContent += `${event.type}: ${event.loaded} bytes transferred\n`;
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/filereader/load_event/index.md:114]:
```suggestion
  eventLog.textContent += `${event.type}：${event.loaded} 传输的字节数\n`;
```

**Review by jasonren0403 — APPROVED** [review]
👍 Thank you

**Pull request merged**